GenumKit/Parsers/StringsFileParser.swift

```diff
@@ -17,12 +31,41 @@ public final class StringsFileParser {
 
   // Localizable.strings files are generally UTF16, not UTF8!
   public func parseStringsFile(path: String) throws {
+    guard let data = NSData(contentsOfFile: path) else {
+      throw StringsFileParserError.FailureOnLoading(path: path)
+    }
+
+    let plist = try NSPropertyListSerialization.propertyListWithData(data, options: NSPropertyListReadOptions.Immutable, format: nil)
+
+    guard let dict = plist as? Dictionary<String, String> else {
+      throw StringsFileParserError.InvalidFormat
+    }
+
     var encoding: NSStringEncoding = NSUTF16StringEncoding
     let fileContent = try NSString(contentsOfFile: path, usedEncoding: &encoding)
+
+    func extractKey(line: String) -> String? {
+      let regex: NSRegularExpression = {
+        do {
+          return try NSRegularExpression(pattern: "^\"([^\"]+)\"[ \t]*=", options: [])
+        } catch {
+          fatalError("Error while building Regular Expression for strings file parsing")
+        }
+      }()
+
+      let range = NSRange(location: 0, length: (line as NSString).length)
+      if let match = regex.firstMatchInString(line, options: [], range: range) {
+        return (line as NSString).substringWithRange(match.rangeAtIndex(1))
+      }
+      return nil
+    }
+
     let lines = fileContent.componentsSeparatedByCharactersInSet(NSCharacterSet.newlineCharacterSet())
 
-    for case let entry? in lines.map(Entry.init) {
-      addEntry(entry)
+    for case let key? in lines.map(extractKey) {
+      if let translation = dict[key] {
+        addEntry(Entry(key: key, translation: translation))
+      }
```

Comment:
Now that we have a `Dictionary<String, String>` of the keys and their translation directly (and don't use the textual representation of lines anymore) we can simply do something like:

```swift
for (key, translation) in dict {
  addEntry(Entry(key: key, translation: translation))
}
```